thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/detection/DetectionTaskInfo.java

```diff
@@ -26,14 +27,17 @@
   private List<DateTime> windowEndTime;
   private AnomalyFunctionDTO anomalyFunctionSpec;
   private String groupByDimension;
+  private DetectionJobType detectionJobType;
```

Comment:
detectionJobType should have a default value.. if you’re sticking to null (or picking a default), initialize it here. Otherwise, if new code tries to run any older tasks, it will fail when converting the json to object